jest.config.js

```diff
@@ -9,7 +9,7 @@ const tsConfigPathMapping = pathsToModuleNameMapper(
 )
 
 const testMatch = ["**/?(*.)test.(js|ts|tsx)", "**/test/index.(js|ts|tsx)"]
-const transformIgnorePatterns = ["node_modules/(?!@messageformat)"]
+const transformIgnorePatterns = ["node_modules/(?!@messageformat|pofile-ts)"]
```

Comment:
I saw that your package is distributed as dual cjs/esm, if everything setup correctly for the dual package jest should consume CJS version and this should not be needed. 

If it's still needed - it means more serious problem here.

 it means commonjs code of linguijs would require a esm package which was not possible until nowdays, but fortunately [possible today](https://socket.dev/blog/require-esm-backported-to-node-js-20). However it will bump a minimal compatible version of nodejs for users to 20.16+, 22.19+, or 24+ which i would consider a breaking change